### DISCUSSION TITLE
Run custom code/script during flight ?

### DISCUSSION THREAD

**Discussion by Ostefanini** [iNavFlight/inav#6454] [Q&A] [answered]
Hello everyone,
I hope it's the right place to ask. Does iNav allow to run custom code / script ? during missions operations for example ?

I'd like to control some leds (ws2812 based as they seamed to be supported by iNav) with my own business logic. Do I need to modify iNav source code to add functionnality or is there any extension mechanism available ?

**Comment by digitalentity** [comment, marked as answer]
Good idea. There is no such functionality yet, but I've opened a feature request https://github.com/iNavFlight/inav/issues/6455 so we can track this properly.
